src/Faithlife.Json/JsonSettings.cs

```diff
@@ -0,0 +1,34 @@
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace Faithlife.Json
+{
+	/// <summary>
+	/// Settings for reading/writing JSON via <see cref="JsonUtility" />.
+	/// </summary>
+	public class JsonSettings
+	{
+		/// <summary>
+		/// Additional converters to use when reading/writing JSON.
+		/// </summary>
+		public IReadOnlyList<JsonConverter> Converters { get; set; }
+
+		/// <summary>
+		/// True if JSON properties without corresponding object properties should throw an exception when reading JSON.
+		/// </summary>
+		/// <remarks>The default is `false`, which ignores any extra JSON properties.</remarks>
```

Comment:
Use `<code>` or just drop the backticks.

Reply:
Ha! It's just so instinctive now that I don't even think about it. But I think [`<c>`](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/xmldoc/code-inline) would be the correct element.